lambdas/credentialissuerreturn/src/main/java/uk/gov/di/ipv/core/credentialissuerreturn/CredentialIssuerReturnHandler.java

```diff
@@ -90,8 +91,22 @@ public APIGatewayProxyResponseEvent handleRequest(
             APIGatewayProxyRequestEvent input, Context context) {
         LogHelper.attachComponentIdToLogs();
         try {
-            auditService.sendAuditEvent(AuditEventTypes.IPV_CRI_AUTH_RESPONSE_RECEIVED);
-            RequestHelper.getIpvSessionId(input);
+
+            String ipvSessionId = RequestHelper.getIpvSessionId(input);
+            String userId = ipvSessionService.getUserId(ipvSessionId);
+
+            AuditEventUser auditEventUser = new AuditEventUser(userId, ipvSessionId);
+            this.componentId =
+                    configurationService.getSsmParameter(
+                            ConfigurationVariable.AUDIENCE_FOR_CLIENTS);
+
+            var ae =
+                    new AuditEvent(
```

Comment:
Why doesn't this Audit event not have extensions like the one produced from `sendIpvVcReceivedAuditEvent()`?

Reply:
Not all audit events have anything to put in the exentions block.